charts/polkadot-secrets/templates/secret-cloudflare-credentials.yaml

```diff
@@ -3,6 +3,7 @@ apiVersion: v1
 kind: Secret
 metadata:
   name: cloudflare-credentials
+  namespace: cert-manager
```

Comment:
What if cert-manager namespace does not exist (yet)? Is it better to place it the default namespace ?

Reply:
Good point, at this stage it should exist, but who knows. Maybe better have a value with a default of `cert-manager`?